todo-app/src/App.js

```diff
@@ -1,10 +1,32 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import footer from './Components/footer'
+import Container from './Components/Container'
+import Input from './Components/Input'
+import './App.css';
+
+const data = require('./todoList.json');
 
 class App extends Component {
+  state = {
+    todoItems: []
+  }
+  componentDidMount(){
+  this.setState({
```

Comment:
This is actually unnecessary. It should be possible to simply set your `state` class property above to your imported `data` directly.